ci-operator/config/medik8s/common/new-release.sh

```
@@ -33,13 +33,13 @@ fi
 cd ${REPO}
 
 # copy only the newest main config (highest OCP version)
-latest_main=$(ls | grep 'main__' | sed -r 's#^.*__##; s#\\.yaml$##' | sort -V | tail -1)
+latest_main=$(ls | grep 'main__' | sed -r 's#^.*__##; s#\.yaml$##' | sort -V | tail -1)
```

Shellcheck (0.11.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)